components/measurement/HeadMetadata.js

```diff
@@ -15,10 +15,10 @@ const HeadMetadata = ({
   const intl = useIntl()
   let description = ''
 
+  const formattedDate = moment.utc(date).format('LL')
```

Comment:
We should reduce using `momentjs` and use the native Intl APIs via `react-intl` more. This can be replaced
```suggestion
  const formattedDate = intl.formatDate(date, { day: 'numeric', month: 'long', year: 'numeric'})
```